modules/shadow/arrow-shadow/build.gradle

```diff
@@ -0,0 +1,46 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.github.jengelman.gradle.plugins:shadow:4.0.0'
+    }
+}
+
+apply plugin: 'com.github.johnrengelman.shadow'
+apply plugin: 'java'
+
+ext {
+    metaArrowVersion = '0.7.3'
```

Comment:
Should this be on the root build.gradle instead?

Reply:
yeah, I'll move it there as part of the `arrow-meta` uber PR